### ISSUE TITLE
Typos in `az monitor metrics alert create -h` text

### ISSUE THREAD

**Issue by johndowns** [Azure/azure-cli#10873]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
There are two typos in the `az monitor metrics alert create -h` help text, in the portion of the help text relating to the `--condition` argument.

**To Reproduce**
Execute `az monitor metrics alert create -h` and observe the following section of the text:

```
    --condition         [Required] : The condition which triggers the rule.
        Usage:  --conditon {avg,min,max,total,count} [NAMESPACE.]METRIC {=,!=,>,>=,<,<=} THRESHOLD
                           [where DIMENSION {includes,excludes} VALUE [or VALUE ...]
                           [and   DIMENSION {includes,excludes} VALUE [or VALUE ...] ...]]

        Dimensions can be queried by adding the 'where' keyword and multiple dimensions can be
        queried by combining them with the 'and' keyword.

        Values for METRIC, DIMENSION and appropriate THRESHOLD values can be obtained from `az
        monitor metrics list-definitions` command.

        Multiple conditons can be specified by using more than one `--condition` argument.
```

**Expected behavior**
The following typos are present in the above text:

1. `Usage:  --conditon` should read `Usage:  --condition` (i.e. `condition` is misspelled).
2. `Multiple conditons` should read `Multiple conditions` (i.e. `conditions` is misspelled).

**Environment summary**
Windows 10, Azure CLI version 2.0.75.

**Additional context**

**Comment by MyronFanQiu** [comment]
@johndowns Thanks~ I will fix it later.

**Comment by MyronFanQiu** [comment]
@johndowns It has been fixed by #11112. Thanks for reporting this issue.
